cmd/relayer/main.go

```diff
@@ -102,7 +102,7 @@ func setLogLevels(c *cli.Context) error {
 		return fmt.Errorf("invalid log level %q", level)
 	}
 
-	_ = logging.SetLogLevel("cmd", level)
```

Comment:
I wasn't sure what string to put here. It's in the `main` package located in the `cmd/relayer` folder. There's a package named `relayer`, so I couldn't use that.